gatsby/src/pages/medical-devices/{BnfClinicalMedicalDeviceInformationGroup.medicalDeviceType__medicalDevice__slug}/{BnfClinicalMedicalDeviceInformationGroup.slug}.tsx

```diff
@@ -125,28 +124,15 @@ const CMPIPage: FC<CMPIPageProps> = ({
 			menu={
 				clinicalMedicalDeviceInformationGroups.length > 1
 					? () => (
-							<StackedNav
```

Comment:
There's also a stacked nav inside {BnfMedicalDeviceType.slug}.tsx that looks like we've missed